tools/gen_api_type_database.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I'm refactoring this in https://github.com/envoyproxy/envoy/pull/9276, at the end this file isn't needed.

Reply:
I kept the script but it now is a simple wrapper around what you did in #9276.